en/uavcan/README.md

```diff
@@ -1,6 +1,10 @@
+<Redirect to="../can/" />
```

Comment:
OK, I added this to redirect the old uavcan doc to can. Was that your ultimate plan?

What I'm trying to work out is what you intend for this doc? I assumed at the end of this process that all uavcan docs would be deleted. So my concern is that this still has information that should be kept. For example the PX4 configuraiton stuff like " `0`: UAVCAN driver disabled." (see below).

Reply:
I moved the rest of the stuff out.

Reply:
Excellent!!!!